spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+require 'capybara/rspec'
+
+RSpec.feature 'Index' do # rubocop:disable Metrics/BlockLength
+  let!(:user) do
+    new_user = User.new(id: 1, name: 'John Doe', email: 'john.doe+2@example.com', bio: "Hello, I'm John!",
+                        posts_counter: 0,
+                        photo: Rack::Test::UploadedFile.new(Rails.root.join('app', 'assets', 'images', 'profile.jpg'),
+                                                            'image/jpeg'))
+    new_user.save
+    new_user
+  end
+
+  let!(:post1) do
+    user.posts.create(title: 'First post', text: 'This is the first post', comments_counter: 0, likes_counter: 0)
+  end
+
+  let!(:post2) do
+    user.posts.create(title: 'Second post', text: 'This is the second post', comments_counter: 0, likes_counter: 0)
+  end
+
+  before do
+    visit '/user/1/post'
+  end
```

Comment:
Good job having the set up for this post index page, however, there are a couple of tests yet to be written such as:

- Tests that assert you can see the user's profile picture.
- Tests that asserts you can see a post's title.
- Tests that asserts you can see some of the post's body.
- Tests that asserts you can see a section for pagination if there are more posts than fit on the view.
- Tests that redirects to the post's show page when you click on a post.


Consider writing the above-mentioned test in order meet up with the milestone requirements